docs/guides/parallel-calls.md

```diff
@@ -0,0 +1,48 @@
+## Performing Sequence and Parallel Calls in Tailcall
+
+Tailcall's API Gateway is crafted to effortlessly manage both sequential and parallel calls, enhancing data retrieval for better performance. This guide will walk you through the steps of making sequential and parallel calls using Tailcall.
```

Comment:
I think what I would like to say here is that the sequence and parallel isn't something that the developer needs to explicitly worry about. Tailcall will do it automatically for you based on the schema defined. Then go from there to talk about a few examples where sequence and parallel is implicitly inferred by Tailcall.